go.sum

```diff
@@ -553,3 +553,5 @@ sqlflow.org/gohive v0.0.0-20191009030413-2b2325ce8b38 h1:AIJwoPuFBMLtiivuvZdqvdv
 sqlflow.org/gohive v0.0.0-20191009030413-2b2325ce8b38/go.mod h1:IudT38uGbK5q+Ztx2AsZDzjc04yBsIOUtIDMh/WSJkk=
 sqlflow.org/gomaxcompute v0.0.0-20191028113124-e214922c510c h1:rbxu1RC8cU8zkEr5CE8Oc6kmvMyXcGoaMr0J5gOARMw=
 sqlflow.org/gomaxcompute v0.0.0-20191028113124-e214922c510c/go.mod h1:6jzsWwKY8h1DcEDyQKeE8FtWKxAB/Xbct9dFfACmpyE=
+vitess.io/vitess v3.0.0-rc.3+incompatible h1:fXrxcjHAr7xHyi+jW03lQYs1viidmMn8QMWB9benLHs=
+vitess.io/vitess v3.0.0-rc.3+incompatible/go.mod h1:h4qvkyNYTOC0xI+vcidSWoka0gQAZc9ZPHbkHo48gP0=
```

Comment:
Remove this dependency?

Reply:
Seems it's automatically added when running `go test`